setup.py

```diff
@@ -2,7 +2,23 @@
 from setuptools import setup
 from setuptools import Command
 
-__version__ = version = VERSION = '0.1'
+
+def _get_version():
+    """Returns the PROTEUS software version
+
+       Returns
+       -------
+       version : str
+            PROTEUS software version
+    """
+
+    version_file = 'src/proteus/version.py'
```

Comment:
Is it possible to update this to be more general, something like, `os.path.join('src','proteus','version.py')` ?

Reply:
@gshiroma Wanted to check that you saw this? Or was there a concern? Thanks!

Reply:
Good catch. Thanks!